node/src/components/storage.rs

```diff
@@ -243,6 +247,18 @@ impl From<NetRequestIncoming> for Event {
         Event::NetRequestIncoming(Box::new(incoming))
     }
 }
+impl From<StorageRequest> for Event {
+    #[inline]
+    fn from(request: StorageRequest) -> Self {
+        Event::StorageRequest(Box::new(request))
+    }
+}
+impl From<AppStateRequest> for Event {
```

Comment:
Please add newlines between `impl` blocks. Could also apply to other files you've modified.